examples/go-kprobe-counter/main.go

```diff
@@ -31,7 +31,7 @@ type Stats struct {
 	Counter uint64
 }
 
-//go:generate bpf2go -cc clang -no-strip -cflags "-O2 -g -Wall" bpf ./bpf/kprobe_counter.c -- -I.:/usr/include/bpf:/usr/include/linux
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -cc clang -no-strip -cflags "-O2 -g -Wall" bpf ./bpf/kprobe_counter.c -- -I.:/usr/include/bpf:/usr/include/linux
```

Comment:
Sorry, one last request. While your PR was pending, a new example was created. Do mind adding your changes to https://github.com/bpfman/bpfman/blob/main/examples/go-uretprobe-counter/main.go#L29